src/pages/api/v1/manufacturers/list.ts

```diff
@@ -0,0 +1,35 @@
+import { routes } from "@seamapi/types/devicedb"
+
+import { withRouteSpec } from "lib/middleware/index.ts"
+import { publicMapManufacturer } from "lib/public-mappings/manufacturer.ts"
+
+export default withRouteSpec({
+  ...routes["/api/v1/manufacturers/list"],
+  methods: ["GET", "POST", "OPTIONS"],
+  auth: "vercel_protection_secret",
+} as const)(async (req, res) => {
+  let filtered_manufacturers = [...req.db.manufacturers]
+
+  if (req.query.integration_status) {
+    filtered_manufacturers = filtered_manufacturers.filter(
+      (manufacturer) =>
+        manufacturer.integration === req.query.integration_status,
+    )
+  }
+
+  res.status(200).json({
+    manufacturers: filtered_manufacturers.map((manufacturer) => {
+      const x_forwarded_seam_base_url = req.headers[
+        "x-forwarded-seam-base-url"
```

Comment:
Is `x-forwarded-seam-base-url` part of the read devicedb api or fake only?

Reply:
it's a header added by seam connect on fake requests

Reply:
Shouldn't this be covered by the `Origin` or `Host` header? Was there a reason to introduce a custom header here?

Reply:
sorry, I meant that seam connect attaches this header to forwarded requests to fakes

those headers don't work because if fakes want to return self-referential URLs they need the full path

e.g. `https://connect.getseam.com/internal/sandbox/[workspace_id]/devicedb/images/view`